### PR TITLE
XWIKI-21550: Remove the border and increase the shadown of the header dropdowns

### PR DESCRIPTION
**Pull request by manuelleduc**
Jira: https://jira.xwiki.org/browse/XWIKI-21550

## Before

image

## After

image

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml [hunk 1]
```diff
@@ -797,7 +800,7 @@ require(['jquery', 'xwiki-meta', 'xwiki-bootstrap-switch'], function ($, xm) {
  * Notifications Header
  ***********************************************/
 #tmNotifications {
-  &amp; > button {
+  &amp; &gt; button {
```
Comment (surli) [inline xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml:803]: It means the rule introduced in https://github.com/xwiki/xwiki-platform/commit/da56c0defbd5d810035ce9164dd38ad08984a709#diff-cf46b5e440697c137f189087169625042ba5cc5ad1d675ddbe82f46d7c2aa703 wasn't working?

Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml:803]: Yes, escaping `>` is not required, but it's done with the XML serializer we are using in the xar format plugin.

**Comment by vaishnavi192** [comment]
is the issue solved or i can work on it? if yes then plz assign

**Comment by manuelleduc** [comment]
> is the issue solved or i can work on it? if yes then plz assign

Hello @vaishnavi192, this is a PR already opened by somebody. We don't use github to track issue, please see https://jira.xwiki.org/ (and https://dev.xwiki.org/xwiki/bin/view/Community/Contributing)

**Review by surli — APPROVED** [review]

**Pull request merged**